event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatcherConfiguration.java

```diff
@@ -16,35 +16,42 @@
 
 package io.sk8s.event.dispatcher;
 
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.cloud.stream.annotation.EnableBinding;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
 import io.fabric8.kubernetes.client.DefaultKubernetesClient;
 import io.fabric8.kubernetes.client.KubernetesClient;
-
 import io.sk8s.core.resource.ResourceEventPublisher;
 import io.sk8s.kubernetes.client.Sk8sClient;
 
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
 /**
  * @author Mark Fisher
  */
 @Configuration
 @EnableBinding
-@EnableConfigurationProperties(EventDispatcherProperties.class)
 public class EventDispatcherConfiguration {
 
 	@Bean
-	public EventDispatchingHandler eventDispatchingHandler(KubernetesClient kubernetesClient) {
-		return new EventDispatchingHandler(kubernetesClient);
+	public EventDispatchingHandler eventDispatchingHandler() {
+		return new EventDispatchingHandler();
 	}
 
 	@Bean
 	public ResourceEventPublisher functionsEventPublisher(Sk8sClient client) {
 		return new ResourceEventPublisher(client.functions());
 	}
 
+@Bean
```

Comment:
this and the following `@Bean` don't appear to be indented

Reply:
Needed to keep track of current nb of replicas and be notified of nb partitions

Reply:
Indentation is off though